open_academy/security/course_security.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="open_academy_course_group_manager" model="res.groups">
+        <field name="name">OpenAcademy / Manager</field>
+    </record>
+
+    <record id="open_academy_course_rule_group_manager" model="ir.rule">
+        <field name="name">Only responsible can modify course</field>
+        <field name="model_id" ref="model_open_academy_course" />
+        <field name="groups" eval="[Command.link(ref('open_academy.open_academy_course_group_manager'))]" />
+        <field name="perm_read" eval="False" />
+        <field name="perm_write" eval="True" /> 
+        <field name="perm_create" eval="False" />
+        <field name="perm_unlink" eval="True" />
+        <field name="domain_force">['|',('responsible_id','=',user.id),('responsible_id','=',False)]</field>
+    </record>
```

Comment:
Why this? 
I don't think you are already in the security chapter as it is one of the last chapters. Please remove all this and follow the documentation.